contribs/gmf/src/layertree/component.js

```diff
@@ -933,7 +936,7 @@ Controller.prototype.toggleSwipeLayer = function(treeCtrl) {
 Controller.prototype.toggleNodeLegend = function(legendNodeId) {
   const div = document.querySelector(legendNodeId);
   if (div) {
-    div.classList.toggle('d-none');
+    div.classList.toggle('show');
```

Comment:
@adube Do you need to make change ? I tested and mobile is actually not working at all to open the legend div if using "show", that why I made this change in the first place... So I don't know where the issue is with that interface.

Is it a class that come from jquery-ui initially ?

Reply:
I'm not sure where it comes from.

That line was recently changed from 'show' to 'd-none', which causes the desktop version to no longer work.

I'll check the mobile version too and see if both are required for both versions to work.

Reply:
Ok, to my understanding the small "cog" icon tries to do 2 different things on mobile and desktop and the way it is currently implemented only one of them can work.

As a potential fix, I'll try to see if I can separate the container as two different elements instead, one for mobile and one for desktop, to make them both work individually.

Reply:
I've made the fix.  Now both apps work properly.

Reply:
Great, looks good to me!